README.md

```diff
@@ -128,8 +128,9 @@ Bu çözüm vesilesi ile TreeSet veri yapısı öğrenilmeli.
 
 [426. Convert Binary Search Tree to Sorted Doubly Linked List](https://github.com/spiralgo/algorithms/issues/15)
 
-Bunu tam da düşündüğümüz gibi, Inoder rekürsif kullanarak çözmüş. Çünkü inorder zâten bir Binary Tree'ye, küçükten büyüğe sıralayarak veriyor.
+Bunu tam da düşündüğümüz gibi, Inorder rekürsif kullanarak çözmüş. Çünkü inorder zâten bir Binary Tree'ye, küçükten büyüğe sıralayarak veriyor.
 Çözüm gayet akla yatkın.
+Tanıma göre, eğer kökün sol alt ağacını sıralarsak, sonra sağ alt ağacını sıralarsak bunların da ortasını kökü koyarsak elimizde bir tür LinkedList olmalı. Bu da özyineli algoritmayı verir.
```

Comment:
I am not sure if we need to sort the tree as it is BST and already sorted in non-decreasing order.
It is why we just need to traverse DPS (in order: Left, Root, Right) and while traversing the tree we should also populate the Doubly Linked List with the same node.

Reply:
I will add the pictures of the question to make it clear.

Reply:
By "sorting", I meant creating a sorted linked list out of it.